wrapper/src/main/java/software/amazon/jdbc/DriverConnectionProvider.java

```diff
@@ -109,6 +111,14 @@ public Connection connect(
       final @NonNull HostSpec hostSpec,
       final @NonNull Properties props)
       throws SQLException {
+    final String strategy = ReadWriteSplittingPlugin.READER_HOST_SELECTOR_STRATEGY.getString(props);
+    if (strategy != null && strategy.equals("roundRobin")) {
```

Comment:
Same thing here about about string literals being on the left side of the equality check.